### DISCUSSION TITLE
Welcome to Disboard Auto Bump Discussions!

### DISCUSSION THREAD

**Discussion by Theblockbuster1** [Theblockbuster1/disboard-auto-bump#8] [General]
## 🤖 Welcome!
  Before you ask a question be sure to [check the issues](https://github.com/Theblockbuster1/disboard-auto-bump/issues)!:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

  If you cannot find a solution feel free to create a new issue, or to discuss it here!

**Comment by The-Thought-Magician** [comment]
Does it work on Android

**Reply by Theblockbuster1** [reply]
You can use [Kiwi Browser](https://play.google.com/store/apps/details?id=com.kiwibrowser.browser) which supports extensions. Note that you must keep the tab in the foreground at all times for it to work. You can use apps to keep the screen on such as [Automate](https://play.google.com/store/apps/details?id=com.llamalab.automate) with this [custom flow](https://llamalab.com/automate/community/flows/38407).

**Comment by ykc504** [comment]
how to drag and drop the file into extensions im in windows 10

**Reply by Theblockbuster1** [reply]
Just follow the instructions, it's designed to be followed in Windows 10. If you don't understand how to drag and drop check Google.

**Comment by dnasod** [comment]
My google extensions wont let me turn it on how do i get around that

**Reply by Theblockbuster1** [reply]
Sometimes if you are signed in with chrome with a School/Work account it may show this 
![image](https://user-images.githubusercontent.com/48650812/113522256-f439e500-9596-11eb-92cf-a1dc82c093d3.png)
You can try disabling it by signing out of Chrome, using a different account as your main Chrome account, or by using a different browser (such as Firefox).

**Comment by Dxniel0** [comment]
is it possible to bump by itself when the computer is turned off?

**Reply by Theblockbuster1** [reply]
Unfortunately not, as the tab must be open, so if the computer is turned off (e.g. Powered off, Hibernation, or Sleep mode), the Auto Bumper wont run. Hope this helps.

**Reply by Dxniel0** [reply]
okay thanks a lot for the answer

**Comment by ptrcao** [comment]
I've found that reCAPTCHA puzzles are being triggered in FireFox for subsequent tabs after the first one - possibly each time the page reloads - thwarting the extension's function?  What can I do to overcome these?

**Comment by ptrcao** [comment]
A separate problem has been encountered in Chrome - whereby multiple tabs will collapse into one over time as the extension is running, making it impossible to run multiple bump tabs at one time.  How can I overcome this?

**Reply by Dxniel0** [reply]
I recommend Opera GX

**Reply by PolishLithuanianCommonwealth** [reply]
I use Opera GX, doesn't fix anything though.

**Reply by PolishLithuanianCommonwealth** [reply]
Problem is still here.

**Reply by Dxniel0** [reply]
okay

**Comment by theoneaykay** [comment]
Disboard showing ads after the bump, how can I over come that?

**Reply by Theblockbuster1** [reply]
Does this still happen? Try using the extension alongside [uBlock Origin](https://ublockorigin.com/)

**Comment by InfernalesUltor** [comment]
Failed to load extension
File
~/Desktop/disboard-auto-bump-master
Error
Manifest file is missing or unreadable
Could not load manifest

**Reply by Theblockbuster1** [reply]
What is the contents of the `disboard-auto-bump-master` folder on your Desktop? It should contain all the original files from the repository, which includes the `manifest.json` file in the root directory.

**Comment by rohitkhatwaniii** [comment]
![Screenshot 2024-09-25 154933](https://github.com/user-attachments/assets/dbae4bf7-f0e6-4144-aede-8111edeaa8d2)

what do i do with this ?

**Comment by alihaiderdar84** [comment]
what do i do with this ?
![image](https://github.com/user-attachments/assets/766cde4c-045c-4a0f-9e98-fc27835f71a8)
